tests/cn/ghost_arguments_type_mismatch.error.c.verify

```diff
@@ -0,0 +1,13 @@
+return code: 1
+tests/cn/ghost_arguments_type_mismatch.error.c:4:3: warning: experimental keyword 'cn_ghost' (use of experimental features is discouraged)
+  cn_ghost boolean b;
+  ^~~~~~~~ 
+[1/2]: foo -- pass
+[2/2]: main -- fail
+tests/cn/ghost_arguments_type_mismatch.error.c:15:11: error: Type error
+  int x = 3;
+          ^ 
+Expression '3'i32' has type 'i32'.
+I expected it to have type 'boolean' because of tests/cn/ghost_arguments_type_mismatch.error.c:1:1:
+int foo(int p)
+~~~~^~~~~~~~~~ 
```

Comment:
This location is also wrong. It should point at the actual parameter declaration.